easybuild/framework/easyblock.py

```diff
@@ -1823,7 +1956,25 @@ def run_step(self, step, methods):
         self.update_config_template_run_step()
         for m in methods:
             self.log.info("Running method %s part of step %s" % ('_'.join(m.func_code.co_names), step))
-            m(self)
+
+            if self.dry_run:
+
+                dry_run_msg("[%s method]" % m(self).__name__, silent=self.silent)
+
+                # if an known possible error occurs, just report it and continue
+                try:
+                    m(self)()
```

Comment:
the method as passed by 'name', so `()` is needed to actually *call* the method, rather than just referring to it, I'll add a comment to clarify that